gatsby-config.js

```diff
@@ -10,34 +14,24 @@ module.exports = {
         path: `${__dirname}/src/images`,
       },
     },
-    // add a gatsby-source-filesystem entry for every article's images
-    // have to filter out empty folders for Zeit
-    ...fs
-      .readdirSync(`${__dirname}/src/pages/blog`)
-      .map((path) => `${__dirname}/src/pages/blog/${path}`)
-      .filter((path) => fs.lstatSync(path).isDirectory())
-      .map((path) => `${path}/img`)
-      .filter((path) => fs.existsSync(path) && fs.readdirSync(path).length > 0)
-      .map((path) => ({
-        resolve: "gatsby-source-filesystem",
-        options: {
-          name: "images",
-          path,
-        },
-      })),
+    // {
+    //   resolve: "gatsby-source-filesystem",
+    //   options: {
+    //     name: "posts",
+    //     path: `${__dirname}/src/pages/blog`,
+    //   },
+    // },
+    
     "gatsby-transformer-sharp",
     "gatsby-plugin-sharp",
+    "gatsby-remark-images",
```

Comment:
Oh very nice, is this new?

Reply:
Yeah I saw that gatsby-plugin-mdx suggested this way (before and inside it)!